src/jit/register_arg_convention.cpp

```diff
@@ -58,7 +58,7 @@ bool InitVarDscInfo::enoughAvailRegs(var_types type, unsigned numRegs /* = 1 */)
 
     unsigned backFillCount = 0;
 
-#ifdef _TARGET_ARM_
+#if defined _TARGET_ARM_ && !defined ARM_SOFTFP
```

Comment:
I don't think this change is necessary

---
In reply to: [57580565](https://github.com/dotnet/coreclr/pull/3929#discussion_r57580565) [](ancestors = 57580565)